api/src/main/java/uk/gov/dvsa/mot/vehicle/read/core/VehicleReadService.java

```diff
@@ -19,7 +19,7 @@
 
     List<DvlaVehicle> findDvlaVehicleByRegistration(String registration);
 
-    List<DvlaVehicle> findDvlaVehicleById(Integer dvlaVehicleId);
+    List<uk.gov.dvsa.mot.trade.api.DvlaVehicle> findDvlaVehicleById(Integer dvlaVehicleId);
```

Comment:
This is unnecessary now since it's imported. Besides that don't forget to squash commits and all's good :).